### PR TITLE
Feat: Instruction page created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
instruction page created for the quiz

### CODE REVIEW

File: css/instruction.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.instruction .btn-start-quiz{
+    text-align: center;
+    background-color: var(--gray-80);
+    border: none;
+    color: var(--gray-00);
+    margin-top: 0.5em;
+    margin-bottom: 1em;
+    position: absolute;
+    top: 0.2em;
+    right: 1em;
```
Comment (partha8) [inline css/instruction.css:34]: any specific reason to use em?

**Review by partha8 — APPROVED** [review]
Looks good, the color variables' naming can be improved

**Review by Megha-Joshi — APPROVED** [review]
Good to merge.

File: css/instruction.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@import url('./style.css');
+main .instruction-categories{
+    padding: 2em 5em;
+    flex-direction: column;
+    align-items: center;
+}
+main .instruction{
+    width: 100%;
+}
```
Comment (vashnavichauhan1825) [inline css/instruction.css:9]: space 1 line gap between classes

File: css/instruction.css (hunk 1 above)
Comment (arjun-computer-geek) [reply css/instruction.css:34]: ye it will increase or decrease according to the current element's font size.

**Pull request merged**